requre/modules_decorate_all_methods.py

```diff
@@ -12,10 +12,10 @@
 from requre.helpers.git.pushinfo import PushInfoStorageList
 from requre.helpers.files import StoreFiles
 
-module_requests = record_requests_for_all_methods
+record_requests_module = record_requests_for_all_methods
```

Comment:
Why we are renaming that? Can you please add some comment to make it clear?

Reply:
I just wanted to have same naming for all modules. and have all of them in one file. It is hard to explain. It is more less about my feeling that it is better to do. I can add there this explanation if it is sufficient for you?